keybase/context.go

```diff
@@ -74,8 +74,8 @@ func newContextCheckCmd(cfg Config, log Log, isCheckCommand bool) *context {
 }
 
 // NewUpdaterContext returns an updater context for Keybase
-func NewUpdaterContext(appName string, pathToKeybase string, log Log, isCheck bool) (updater.Context, *updater.Updater) {
-	cfg, err := newConfig(appName, pathToKeybase, log)
+func NewUpdaterContext(appName string, pathToKeybase string, log Log, isCheck bool, ignoreSnooze bool) (updater.Context, *updater.Updater) {
```

Comment:
Not ideal to have a bunch of bools piling up on top of each other here. It's quite easy to mix them up. Do all 4 combinations make sense?